service/http.proxy/src/test/scripts/org/kaazing/gateway/service/http/proxy/http.proxy.malformed.missing.host.identifier.rpt

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# client
+#
+
+connect tcp://localhost:8110
+connected
+
+write "GET /get HTTP/1.1" "\r\n"
+write "\r\n"
+
+read "HTTP/1.1 400 Bad Request" "\r\n"
+
+#
+# server
+#
+
+accept tcp://localhost:8080
+accepted
+connected
```

Comment:
@vstratan You can just leave accept for the origin server, as the http.proxy is not supposed to connect to it, and the test will pass.